actionpack/CHANGELOG.md

```diff
@@ -1,12 +1,20 @@
-*   Allow relative redirects when `raise_on_open_redirects` is enabled
+* Add the ability to use custom logic for storing and retrieving CSRF tokens.
+
+  By default, the token will be stored in the session.  Custom classes can be
+  defined to specify arbitrary behaviour, but the ability to to bypass
+  storing tokens in an empty session is built in.
+
+    * Andrew Kowpak *
+
+* Allow relative redirects when `raise_on_open_redirects` is enabled
 
     *Tom Hughes*
 
-*   Allow Content Security Policy DSL to generate for API responses.
+* Allow Content Security Policy DSL to generate for API responses.
```

Comment:
Please don't edit old entries, if there's a formatting issue it's best to fix it out of this PR.

Reply:
The existing formatting in this file is correct -- I'm guessing this was just an overzealous editor autoformat-on-save.

Reply:
Yeah... this was totally my editor during the merge.  I'll fix this.